shared/js/devtools/panel.es6.js

```diff
@@ -260,3 +274,12 @@ displayFilters.forEach((input) => {
         document.querySelectorAll('tr').forEach(setRowVisible)
     })
 })
+
+/**
+ * Observes the dev settings for resizing to ensure the table head sticks correctly to the bottom of the settings.
+ */
+const settingsResizeObserver = new ResizeObserver(function (entries) {
```

Comment:
As mentioned on MM I think we can avoid this with using a scrollable tbody or similar solutions that are pure CSS.